### ISSUE TITLE
Contact Form Submission - Other (Silver - Greedy Algorithms with Sorting)

### ISSUE THREAD

**Issue by maggieliu05** [cpinitiative/usaco-guide#4042]
Someone submitted the contact form!

**URL**: https://usaco.guide/silver/greedy-sorting?lang=cpp
**Module**: Silver - Greedy Algorithms with Sorting
**Topic**: Other
**Message**: 
Topic: Recommendation
I just completed bronze and almost all the problems in it except some "easy" ones which I've saved for leisure. I actively contribute solutions for problems which are teaching something, with explanations.
First of all, this is a great website and Thanks to every single one of you, I love y'all.
I wanted to know if I should start silver or practice in CF until I reach like 1300/1400 rating, I'm currently at around 1100. Or should I start silver and just give contests without explicit practice in CF.

TLDR:
Just completed Bronze, should I jump to Silver or keep practicing till I reach !400 on CF. My main aim is a good rating in CF and not USACO as Im past high school.

**Comment by bqi343** [comment]
idk, doing whichever seems more interesting should be ok
